src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,3158 @@
+                    />
+                  </g>
+                </svg>
+              </td>
+              <td
+                class="css-1ai0fqu"
+                role="gridcell"
+              >
+                <svg
+                  class="chakra-icon css-1n9hd9c"
+                  focusable="false"
+                  viewBox="0 0 14 14"
+                >
+                  <g
+                    fill="currentColor"
+                  >
+                    <polygon
+                      points="5.5 11.9993304 14 3.49933039 12.5 2 5.5 8.99933039 1.5 4.9968652 0 6.49933039"
+                    />
+                  </g>
+                </svg>
+              </td>
+              <td
+                class="css-1ai0fqu"
+                role="gridcell"
+              >
+                <svg
+                  class="chakra-icon css-1n9hd9c"
+                  focusable="false"
+                  viewBox="0 0 14 14"
+                >
+                  <g
+                    fill="currentColor"
+                  >
+                    <polygon
+                      points="5.5 11.9993304 14 3.49933039 12.5 2 5.5 8.99933039 1.5 4.9968652 0 6.49933039"
+                    />
+                  </g>
+                </svg>
+              </td>
+            </tr>
+            <tr
+              class="css-0"
+              role="row"
+            >
+              <td
+                class="css-1l7w1il"
+                role="gridcell"
+              >
+                SIEM integration
+              </td>
+              <td
+                class="css-1ai0fqu"
+                role="gridcell"
+              />
+              <td
+                class="css-1ai0fqu"
+                role="gridcell"
+              >
+                <svg
+                  class="chakra-icon css-1n9hd9c"
+                  focusable="false"
+                  viewBox="0 0 14 14"
+                >
+                  <g
+                    fill="currentColor"
+                  >
+                    <polygon
+                      points="5.5 11.9993304 14 3.49933039 12.5 2 5.5 8.99933039 1.5 4.9968652 0 6.49933039"
+                    />
+                  </g>
+                </svg>
+              </td>
+              <td
+                class="css-1ai0fqu"
+                role="gridcell"
+              >
+                <svg
+                  class="chakra-icon css-1n9hd9c"
+                  focusable="false"
+                  viewBox="0 0 14 14"
+                >
+                  <g
+                    fill="currentColor"
+                  >
+                    <polygon
+                      points="5.5 11.9993304 14 3.49933039 12.5 2 5.5 8.99933039 1.5 4.9968652 0 6.49933039"
+                    />
+                  </g>
+                </svg>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <h3
+        class="chakra-heading css-1uy7oy0"
+      >
+        Non-cluster hosts
```

Comment:
Recently changed...

"Restrict traffic.." and "automatic host endpoints" for Calico Cloud should be false.